src/content/docs/en/reference/cli-reference.mdx

```diff
@@ -492,11 +506,14 @@ The inline config passed in this object will take highest priority when merging
 
 <p>
 
-**Type:** `"development" | "production"`<br />
-**Default:** `"development"` when running `astro dev`, `"production"` when running `astro build`
+**Type:** `string`<br />
+**Default:** `"development"` when running `astro dev`, `"production"` when running `astro build`<br />
+<Since v="5.0.0" />
 </p>
 
-The mode used when building your site to generate either "development" or "production" code.
+The mode used when developing or building your site. It's passed to Vite that affects the value of `import.meta.env.MODE` and how `.env` files are loaded, which also affects the values of `astro:env`. See the [environment variables page](/en/guides/environment-variables/) for more details.
```

Comment:
```suggestion
The mode used when developing or building your site (e.g. `"production"`, `"testing"`).

This value is passed to Vite using [the `--mode` flag](#--mode-string) when the `astro build` or `astro dev` commands are run to determine the value of `import.meta.env.MODE`.This determines which `.env` files are loaded, and therefore the values of `astro:env`. See the [environment variables page](/en/guides/environment-variables/) for more details.
```
I might be making things up, but is this correct? 😄

Reply:
Looks right to me!